ui/components/app/transaction-decoding/components/ui/copy-raw-data/copy-raw-data.component.js

```diff
@@ -0,0 +1,38 @@
+import React, { useContext } from 'react';
```

Comment:
This is for the "Copy raw transaction data" component in the hex tab:

![Screenshot from 2021-12-01 09-42-44](https://user-images.githubusercontent.com/7499938/144240612-020434c8-3a4f-42b8-8931-118de59272d8.png)